docs/release/release_0_4_17.md

```diff
@@ -12,8 +12,40 @@ https://github.com/napari/napari
 
 ## Highlights
 
-- Feature: Minimum blending (#4875)
-- Add option to edit second shortcut in settings (#5018)
+This release is focused on documentation improvements and bug fixes, with few changes to napari’s API.
+We picked out a few highlights, but keep reading below for the full list of changes!
+
+### Minimum blending for a better light theme experience
+
+We added a new minimum blending option in #4875 that is particularly useful for those
+using the napari light theme.
+See [examples/minimum_blending.py](https://github.com/napari/napari/blob/main/examples/minimum_blending.py) for an example.
+
+### Option to edit second shortcut in settings
+
+TODO
+
+### `napari.imshow` as a convenient one-liner
+
+TODO
```

Comment:
@kcpevey : could you add one or two sentences (and maybe a short code example) to highlight this work?

Reply:
A convenience method to load image data and return both a Viewer and the Layer (also accepts an existing viewer).
```
data = np.random.random((10, 15))
viewer, layer = napari.imshow(data)
```

Reply:
Thanks will add this soon.